apple-pie/styles/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+    margin-top: 0%;
```

Comment:
To get rid of the white bars around the background image instead of `margin-top: 0%;` that will only take out the top margin, you can do `margin: 0px` and in this way you will take out all margins (top, right, bottom and left).